tgui/packages/tgui/interfaces/Reflector.tsx

```diff
@@ -0,0 +1,242 @@
+                          rotation_angle: 0,
+                        })
+                      }
+                    />
+                  </Table.Row>
+                  <Table.Row>
+                    <Box px={0.75}>
+                      <Icon
+                        name="angle-double-up"
+                        size={1.66}
+                        rotation={rotation_angle}
+                        mb={1}
+                      />
+                    </Box>
+                  </Table.Row>
+                  <Table.Row>
+                    <Button
+                      icon="arrow-down"
+                      mb={1}
+                      onClick={() =>
+                        act('rotate', {
+                          rotation_angle: 180,
+                        })
+                      }
+                    />
+                  </Table.Row>
+                </Table.Cell>
+                <Table.Cell>
+                  <Table.Row>
+                    <Button
+                      icon="arrow-right"
+                      iconRotation={-45}
+                      mb={1}
+                      onClick={() =>
+                        act('rotate', {
+                          rotation_angle: 45,
+                        })
+                      }
+                    />
+                  </Table.Row>
+                  <Table.Row>
+                    <Button
+                      icon="arrow-right"
+                      mb={1}
+                      onClick={() =>
+                        act('rotate', {
+                          rotation_angle: 90,
+                        })
+                      }
+                    />
+                  </Table.Row>
+                  <Table.Row>
+                    <Button
+                      icon="arrow-right"
+                      iconRotation={45}
+                      mb={1}
+                      onClick={() =>
+                        act('rotate', {
+                          rotation_angle: 135,
+                        })
+                      }
+                    />
+                  </Table.Row>
+                </Table.Cell>
+              </Table>
+            </Section>
+          </Stack.Item>
+          <Stack>
+            <Section title="Angle" textAlign="center" fill>
+              <LabeledControls>
+                <LabeledControls.Item ml={0.5} label="Set rotation">
+                  <NumberInput
+                    value={rotation_angle}
+                    unit="degrees"
+                    minValue={0}
+                    maxValue={359}
+                    step={1}
+                    stepPixelSize={1}
+                    onDrag={(e, value) =>
+                      act('rotate', {
+                        rotation_angle: value,
+                      })
+                    }
+                  />
+                </LabeledControls.Item>
+              </LabeledControls>
+              <Stack direction="row" grow={1}>
```

Comment:
```suggestion
              <Stack fill>
```
Grow does not work on stack (only stack.item,I think this meant to use fill
Row is the default direction so this isn't needed